src/framework/filterControl/_filterItem.scss

```diff
@@ -45,3 +47,11 @@
         opacity: 1;
       }
     }
+
+.filterFormDropdown--editing {
```

Comment:
Bit weird to find `filterFormDropdown` instead inside its corresponding scss file.